ui/app/store/actions.js

```diff
@@ -2241,6 +2241,13 @@ export function setSwapsLiveness(swapsFeatureIsLive) {
   }
 }
 
+export function setSwapsQuoteRefreshTime(refreshTime) {
```

Comment:
This looks like more dead code - it was only called in that `swaps` slice function that was also removed. The `setSwapsQuoteRefreshTime` isn't a part of the background API anymore either.